clang/test/SemaSYCL/deferred-diagnostics-aux-builtin.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown-sycldevice -fsycl -fsycl-is-device -aux-triple x86_64-unknown-linux-gnu -verify -fsyntax-only  %s
+//
+/*
+ This test is to verify that deferred diagnostics are emitted whenever there is an AUX target builtin function inside device code.
```

Comment:
Don't use C style comments in this file.  Also, comments are overly wordy to the point of being distracting.